_includes/getting-started.html

```diff
@@ -318,24 +304,27 @@ <h3>Helm Chart:</h3>
 
             <h4 style="font-size: 1.5em;">Install on Kubernetes using Helm:</h4>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 60px; line-height: 30px;"><code style="position: absolute;"> kubectl create namespace meshery
-                helm install meshery --namespace meshery install/kubernetes/helm/meshery </code>
+              <p class="code-box" style="height: 90px; line-height: 30px;"><code style="position: absolute;"> kubectl create namespace meshery
+                helm repo add meshery https://meshery.io/charts/
+                helm install meshery meshery/meshery -n meshery </code>
               </p>
                 <!-- copy to clipboard -->
                 <a class="btn tooltip" style="position: absolute; top:0.10rem; right: 0.5rem; font-size: 22px;" data-clipboard-target="#start-helm-v3"
                   data-clipboard-text="kubectl create namespace meshery
-                  helm install meshery --namespace meshery install/kubernetes/helm/meshery" onmouseout="resetCopyText(this)">
+                helm repo add meshery https://meshery.io/charts/
+                helm install meshery meshery/meshery -n meshery" onmouseout="resetCopyText(this)">
                   <i class="far fa-copy"></i>
                   <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 3rem; left: -20px;">Copy to clipboard</span>
                 </a>
             </div>
             <p style="font-size: 1em;">Not a Helm user? Use the Meshery manifests directly.</p>
```

Comment:
If we are not showing the `manifests` command then, why show this line?

Reply:
That is for FYI to the end user.